packages/nodes-base/nodes/Zammad/descriptions/UserDescription.ts

```diff
@@ -0,0 +1,658 @@
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a user',
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete a user',
+			},
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Retrieve a user',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				description: 'Retrieve all users',
+			},
+			{
+				name: 'Get Self',
+				value: 'getSelf',
+				description: 'Retrieve currently logged-in user',
+			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update a user',
+			},
+		],
+		default: 'create',
+	},
+
+	// ----------------------------------
+	//             fields
+	// ----------------------------------
+	{
+		displayName: 'Email Address',
+		name: 'email',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+				operation: [
+					'create',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'User Name/ID',
+		name: 'id',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'loadUsers',
+		},
+		description: 'User to update. Choose from the list or specify an ID using an <a href="https://docs.n8n.io/nodes/expressions.html#expressions">expression</a>.',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+				operation: [
+					'update',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'User Name/ID',
+		name: 'id',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'loadUsers',
+		},
+		description: 'User to delete. Choose from the list or specify an ID using an <a href="https://docs.n8n.io/nodes/expressions.html#expressions">expression</a>.',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+				operation: [
+					'delete',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'User Name/ID',
+		name: 'id',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'loadUsers',
+		},
+		description: 'User to retrieve. Choose from the list or specify an ID using an <a href="https://docs.n8n.io/nodes/expressions.html#expressions">expression</a>.',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+				operation: [
+					'get',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+				operation: [
+					'create',
+				],
+			},
+		},
+		default: {},
+		placeholder: 'Add Field',
+		options: [
+			{
+				displayName: 'Active',
+				name: 'active',
+				type: 'boolean',
+				default: false,
+			},
+			{
+				displayName: 'Address',
+				name: 'addressUi',
+				type: 'fixedCollection',
+				default: {},
+				options: [
+					{
+						displayName: 'Address Details',
+						name: 'addressDetails',
+						values: [
+							{
+								displayName: 'City',
```

Comment:
UX-wise there is no "logical" order: "While the order you have mentioned is typical for a specific group of countries, it may be totally different in other ones."
https://ux.stackexchange.com/questions/44626/order-of-address-city-country-in-form